runtime/vm.go

```diff
@@ -634,11 +642,36 @@ func (v *VM) run() {
 				v.sp++
 			}
 
+		case compiler.OpSpread:
+			sp := v.sp - 1
+			obj := v.stack[sp]
+			if SpreadValues, ok := obj.(objects.Spreadable); ok {
+				v.stack[sp] = &objects.SpreadValues{Value: SpreadValues.Spread()}
```

Comment:
we need to track down the number of allocations: `v.allocs`

Reply:
would this be all that is needed?
```golang
v.allocs--
if v.allocs == 0 {
    v.err = ErrObjectAllocLimit
    return
}
```

Reply:
Technically speaking, yes because we're allocating another object. (And, that's why I commented it here.) But, now that I think about it again, from a user standpoint, I'm not sure if it makes sense using `...` operator involves a new object allocation.